source/docs/type-properties.md

```diff
@@ -5,6 +5,110 @@ next: /docs/boolean-type.html
 ---
 # Common Type Properties
 
-> No content yet
+<a name="desc">
+## desc / description
+
+The `desc` (or `description`) property controls the text displayed immediately the right
+of the option or argument in the generated help text.
+
+If not specified, the description will be blank.
+
+```js
+sywac.boolean('--confirm', {
+  desc: 'skip confirmation prompt'
+});
+```
+```bash
+Options:
+  --confirm  skip confirmation prompt                                  [boolean]
+```
+
+See also the [hints](#hints) property.
+
+<a name="group">
+## group
+
+The `group` option allows you to organize options into multiple sections in the generated
+help text. By default, all options are in the section titled `Options:`.
```

Comment:
Perhaps we should mention the default `Commands:` and `Arguments:` groups as well:

> By default, all commands go in a `Commands:` section, all positional arguments go in an `Arguments:` section, and all flagged options go in an `Options:` section.